ExampleMod/Backgrounds/ExampleCloud.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.Utilities;
+using static Terraria.Cloud;
+using static Terraria.WorldGen;
+
+namespace ExampleMod
+{
+	class ExampleCloud
+	{
+		public static UnifiedRandom Rand = new UnifiedRandom();

```

Comment:
Any reason you're making a new random instead of using an existing one? (eg. Main.rand)

Reply:
cause I was stupid at the time and didn't know how it worked cause I was always making my own, I know now, I honestly made this a little fast and should've worked on it more than I did, sorry. I'll try to get back to it some time